coderadar-core/src/main/java/io/reflectoring/coderadar/projectadministration/domain/Commit.java

```diff
@@ -8,15 +9,15 @@
 @Data
 public class Commit {
 
-  private long id;
+  @JsonIgnore private long id;
```

Comment:
I know we discussed this part. I just wanted to ask if you've tried doing this with mapping? Just curious if it's noticeably slower. If it performs exactly the same, then maybe we could consider mapping.